cg-spec/editors_draft.html

```diff
@@ -88,7 +88,6 @@
         repoURL: "w3c/rdf-star",
         branch: "main",
       },
-      wgPublicList: "public-rdf-star",
```

Comment:
Not supposed to be removed?

Reply:
Strange. I don't think I have removed this line explicitly :-/
Anyways, I have restored it (commit https://github.com/w3c/rdf-star/pull/115/commits/b7f7c2fb5e81985c99b0c5dfccc75fcc148d15fd)